app/models.py

```diff
@@ -357,6 +357,31 @@ class Fido(Base, ModelMixin):
     __table_args__ = (sa.Index("ix_fido_user_id", "user_id"),)
 
 
+class AbuserData(Base, ModelMixin):
+    __tablename__ = "abuser_data"
+
+    user_id = sa.Column(sa.Integer, nullable=False)
+    encrypted_bundle = sa.Column(sa.LargeBinary(), nullable=False)
+
+    __table_args__ = (sa.Index("ix_abuser_data_id", "id"),)
+
+
+class AbuserLookup(Base, ModelMixin):
+    __tablename__ = "abuser_lookup"
+
+    hashed_address = sa.Column(sa.String(), nullable=False)
+    blob_id = sa.Column(
```

Comment:
```suggestion
    abuser_data_id = sa.Column(
```
We always use `table_name_id` as foreign keys to make it clear the table they reference.

Reply:
Also we need to support the same address having more than one abuse report since an email can exist as a mailbox for several users.